script/test

```diff
@@ -107,7 +107,7 @@ generate_key_and_certificate ${TEST_PRIVATE_KEY} ${TEST_CERTIFICATE}
 generate_key_and_certificate ${OTHER_TEST_PRIVATE_KEY} ${OTHER_TEST_CERTIFICATE}
 
 echo "==> Starting punica(-s)..."
-REGULAR_PUNICA_PID=$(run_punica "regular")
+REGULAR_PUNICA_PID=$(run_punica "regular" "-d ${PROJECT_ROOT_DIR}/tests/rest/tmp_database.json")
```

Comment:
Master branch has a database file for function database_load_file code coverage, while /devices rest tests makes changes to the database file. I don't want to use the same file since that would cause different results when calling tests multiple times in a row. A solution could be to make a temporary copy, and use it for both test cases.

Reply:
that sounds good, although simply using valid database on regular and temporary on secure would work out too.